properties/P000141.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: P000141
+slug: compactly-generated-t2-maps
+name: Generated by compact Hausdorff maps
+aliases:
+    - Compactly generated
+    - k-space
+    - Kelley space
```

Comment:
Where does Kelley space come from?  As far as I know, the notion of compactly generated in Kelley's book of 1955 was "coherent with the collection of closed compact subspaces", which is different.

Reply:
Took this from https://ncatlab.org/nlab/show/compactly+generated+topological+space (also uses "kaonic space")

Reply:
Checking the pdf linked from the nlab article, Kelley space is defined as Hausdorff space with P140.  I don't think we should even mention this in pi-base.